<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting_bandito_widget</class>
 <widget class="QWidget" name="Setting_bandito_widget">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>372</width>
    <height>320</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignTop">
    <widget class="QFrame" name="settings_qFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>310</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>310</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="ipSource">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="ip_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>IP Source:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ip_source_lineE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ipDestination">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="ip_destination_label">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>IP Destination:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ip_destination_lineE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="port">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="port_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="port_lineE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="login">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="login_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Login:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="login_lineE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="password">
        <property name="spacing">
         <number>5</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QLabel" name="pass_label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pass_lay">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="pass_lineE">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="show_pass_checkB">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="button_layout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="connect_toolB">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="disconnect_toolB">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="save_toolB">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ping_toolB">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Ping</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="status_light">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QFrame" name="signal_frame">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="status_lineE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>35</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
